docs-2.0/7.data-security/2.backup-restore/1.what-is-br.md

```diff
@@ -0,0 +1,69 @@
+# What is Backup & Restore
+
+Backup & Restore (BR in short) is a Command-Line Interface (CLI) tool for you to back up data of graph spaces of Nebula Graph and to restore data from the backup files.
+
+## Implementation
+
+### Backup
+
+To back up data, the BR sends a backup request to each metad process to trigger the backup process as follows:
+
+1. The SSH login from the BR to the meta and the storage servers is verified. Besides, if remote store such as Amazon S3 or Alibaba Cloud OSS is necessary, their client installation and configuration is verified.
+2. The BR sends a request to create backup files.
+3. The metad process is locked.
+4. The metad process blocks writes to the specified graph spaces on the storaged process.
+5. The metad process sends a request to storaged processes for the snapthot file names.
+6. The metad process scans local RocksDB files and outputs SST files.
+7. The metad process cancels blocking writes to the graph spaces on the storaged process.
+8. The metad process sends response to the BR with the metad data including the thrift format, partition information of the graph spaces, and the log commit ID of each partition, and the snapshot information including the catalog of the snapshots of each storaged process, their SST file names of the meta server, and the backup file names.
```

Comment:
Does **The metad process sends response to the BR with the metad data** means **The metad process sends response to the BR by using the metad data**?
I’m confused with the **with** here.